docs/docs/explanations/why-we-built-kurtosis.md

```diff
@@ -1,17 +1,21 @@
 ---
```

Comment:
@leeederek @mieubrisse  I would opt to just remove this page entirely. When we're ready to post a roadmap communicating product vision publicly, we can add the content back into the docs in a way thats directly relevant to users (they will be able to internalize where we're going, give feedback). 

Given new philosophy of being **specific** and **current** with our product, I'd say this page no longer fits into the docs (could be good for a blog post)

Reply:
Caught up with Galen - we concluded that this content is valuable to leave as is to help convey the vision/purpose of Kurtosis